010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
+- **File attachments**: Store and retrieve files up to 1GB, including images, videos, and audio files. The files are stored in object storage and automatically cahced on a global CDN for fast delivery. From the application point of view, however, it's just another column in the database.
+- **Full-text search**: Data is automatically replicated into a full-text search engine, making it easy to build search experiences, supporting fuzzy search, weights, and relevancy boosters.
+- **Vector search**: Easily store embeddings and implement similarity search to power your AI applications.
+- **Admin UI**: A web-based console to manage your data, including schema and data.
+- **SDKs**: Xata comes with vertically integrated SDKs for popular programming languages, including TypeScript, JavaScript, Python, and Go. In addition, Xata works with any PostgreSQL client or ORM.
 
-This type of service has several benefits:
+In order to provide these features, Xata uses other data stores in addition to PostgreSQL, namely Elasticsearch for full-text search and aggregations, S3 for file attachments, and the Cloudflare CDN for file delivery and image transformations. The Postgres logical replication events are used to keep these data stores in sync with the main database.
 
-- it simplifies building applications, thanks to its vertical integration and serverless capabilities
-- it offers more data-related functionality than a typical database, for example, free-text-search with relevancy controls, and more advanced analytics
-- it grows with you, in both scale and required functionality because it is based on proven data technologies, each best-in-class for the problems they solve.
+Access to the underlying data is provided in two main ways: via the PostgreSQL wire protocol, which is mediated by our SQL Proxy, and via our SDKs, which provide a higher-level interface to the data.
 
-Currently, the Xata service uses PostgreSQL as the data store, Elasticsearch as the search/analytics engine, and Kafka for storing the logical replication events. It also offers support for edge-caching. You can read more about how Xata works in this document and a more detailed explanation of the serverless aspect in this [blog post](https://xata.io/blog/what-is-a-serverless-database).
+![PostgreSQL Data Platform](images/postgresql_data_platform.png)
 
-In the future, we plan to also add built-in support for centralized caching (for example, Redis), block storage, better support for sharding between DBs, automatic global distribution, and so on.
+Individual and indie developers like Xata because it is easy to get started, has a generous free-tier, offers a nice admin UI, and solves common problems in a developer centric way (e.g. file attachments, full-text search, vector search).
```

Comment:
```suggestion
Individuals and indie developers like Xata because it is easy to get started, has a generous free-tier, offers a nice development experience, and solves common problems in a developer-first way (e.g. file attachments, full-text search, vector search).
```